platforms/minigame/platforms/taobao-mini-game/wrapper/builtin/WebSocket.js

```diff
@@ -55,10 +55,26 @@ export default class WebSocket {
     my.onSocketOpen(this._onOpen)
 
     this._onMessage = (res) => {
-      if (res && res.data && res.isBuffer) {
-        res.data = _utils.base64ToArrayBuffer(res.data)
+      function deepCopy(obj) {
+        if (typeof obj !== 'object' || obj === null) {
```

Comment:
Use `obj == null` which will also support `undefined`.
Should it also need to check `undefined` value?

Reply:
Yes, need to check undefined value.
![image](https://github.com/user-attachments/assets/bac4424f-1c67-4ad5-b045-f663911b16ae)